.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+< A brief description of the pull request >
+
+---
+
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| New feature?  | yes/no
+| BC breaks?    | yes/no
+| Deprecations? | yes/no
+| Tests pass?   | yes/no
+
+---
+
+## Before
+
+< The current status >
+
+## After
+
+< What will be changed >
+
+## Screenshots
+
+< If required. [LICEcap](http://www.cockos.com/licecap/) for Mac or SilentCast for Linux allow you to great GIFs from screen capture >
+
+## Comments
```

Comment:
In my experience "Overview", "Before" and "After" are all non-technical, covering the "what" of the PR. Even if i'm a developer, before you throw at me technical details I need to have an understanding "as a user" of what this PR does (of course if the PR is not technical-only)

The "Technical Details" section would be then where (if needed!) the PR author would go into the details of the "how". Things like architectural changes, new classes/components, changes to the build process, and whatever else you might need to document that other developers would find relevant